billmania/roboquest_ui
# Transfer UPDATE payload to the updater

Previously WebServer.update_software() sent a dummy payload to the updater. Now, the payload from the UI is transferred instead of the dummy version.

Also defined an RQ_PARAMS.UPDATE_FIFO to replace the hard-coded string in web_server.js.

See Issue #45 

Test requires the fix from Issue #44 
1. create the FIFO locally
2. cat /tmp/update_fifo
3. start the rq_ui container locally
4. click the update robot software button in the UI
5. verify the string read from the FIFO by step 

The string
```
{"timestamp":"1693230124","version":"1","action":"UPDATE","args":"UI"}
```
is written to the FIFO.
